modules/ztp-enabling-assisted-installer-service-on-bare-metal.adoc

```diff
@@ -3,20 +3,22 @@
 // * scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc
 
 [id="enabling-assisted-installer-service-on-bare-metal_{context}"]
-= Enabling the assisted service and updating AgentServiceConfig on the hub cluster
+= Enabling the assisted service
 
-{rh-rhacm-first} uses the assisted service to deploy {product-title} clusters. The assisted service is deployed automatically when you enable the MultiClusterHub Operator with Central Infrastructure Management (CIM). When you have enabled CIM on the hub cluster, you then need to update the `AgentServiceConfig` custom resource (CR) with references to the ISO and RootFS images that are hosted on the mirror registry HTTP server.
+{rh-rhacm-first} uses the assisted service to deploy {product-title} clusters. The assisted service is deployed automatically when you enable the MultiClusterHub Operator on {rh-rhacm-first}. After that, you need to configure the `Provisioning` resource to watch every `Namespace` and to update the `AgentServiceConfig` custom resource (CR) with references to the ISO and RootFS images that are hosted on the mirror registry HTTP server.
```

Comment:
I would maybe try to keep the use of literals the same in the PR.  On line 20, you have `Provisioning` resource and `...to watch all namespaces`. I'd make like 8 and line 20 consistent across their use of back-ticks. For me, line 20's language of `watch all namespaces` reads cleaner than line 8's `every Namespace`.